var/httpd/htdocs/js/Core.Agent.DynamicFieldDBSearch.js

```diff
@@ -258,13 +258,21 @@ Core.Agent.DynamicFieldDBSearch = (function(TargetNS) {
                     /TicketID=(\d+)/.exec(document.URL);
                     TicketID = RegExp.$1;
 
+                    // for usage in process context
+                    var LiElementId = $Element.parents().closest('li.Activity').attr('id');
+                    var DynamicFieldNameQuery = DynamicFieldName;
+                    if (typeof LiElementId !== 'undefined') {
```

Comment:
Did you try that for the normal reply (id=FollowUp) in CustomerTicketZoom? It also has the Activity-class, but the substitution is not needed. Maybe it is necessary to include the "match" as "test" in the if condition.